faqs/galaxy/analysis_reporting_issues.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Reporting Issues and Feature Requests
+area: analysis
+box_type: tip
+layout: faq
+contributors: [jennaj, garimavs]
+---
+
+- For reporting **Usage Problems**, related to tools and functions, head to the [Galaxy Help](https://help.galaxyproject.org/) site.
+    - **Red error datasets**:
+        - To resolve it you might be asked to send a bug report.
+        - Click on the green bug icon within one of the red error datasets to start the submission process 
+    - **Green unexpected-result datasets**:
+        - To resolve it you may be asked to send in a shared history link and possibly a shared workflow link. 
+        - Email our team directly on the private mailing list galaxy-bugs@lists.galaxyproject.org. Include the share link(s) and note the problem dataset numbers involved.
+    - **Functionality problems**:
+        - Using [Galaxy Biostars](https://galaxyproject.org/support/biostar/) is the best way to get help in most cases.
```

Comment:
Galaxy Biostars does not exist anymore (I don't even know what it was!). I guess redirecting users to the Galaxy Help forum would do.